### PR TITLE
[7] GET /todos

### PR DESCRIPTION
**Pull request by pdany1116**
#### Description

Implement GET /todos request.

GET Request on /todos should return an array of Todo resources. Empty array should be returned if no Todo resource exists.

No request body needed.

HTTP Statuses:
On success -> 200

## Steps to Test or Reproduce

1. Perform GET /todos.
2. Status should be 200.
3. Empty array should be returned as body.

### CODE REVIEW

File: spec/requests/todos_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+require 'rails_helper'
+
+RSpec.describe 'Todos', type: :request do
+  describe 'GET /todos' do
+    subject(:get_todos) { get '/todos' }
+
+    context 'with no todos added' do
+      let(:empty_array) { [] }
+
+      before { get_todos }
```
Comment (cazacutudor) [inline spec/requests/todos_spec.rb]: As we discussed, here's the suggestion for formating the specs.

https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Comment (pdany1116) [reply spec/requests/todos_spec.rb]: c92c3ccb8168843d88483ffa75f81eef62f71b66

File: spec/requests/todos_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    context 'with 5 todos added' do
+      before { create_list(:todo, 5) }
+
+      it 'returns 200' do
+        get_todos
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'returns array with 5 todos' do
+        get_todos
```
Comment (cazacutudor) [inline spec/requests/todos_spec.rb:33]: A new line may help here, to differentiate between the act and assertion.

**Review by cazacutudor — APPROVED** [review]

File: spec/requests/todos_spec.rb (hunk 2 above)
Comment (pdany1116) [reply spec/requests/todos_spec.rb:33]: 3a09cea4e64ca0de48aeb45de3c8ff106c228926

**Pull request merged**